protocols/imap/src/main/java/org/apache/james/imap/message/response/ListRightsResponse.java

```diff
@@ -54,25 +55,25 @@ public List<MailboxACL.Rfc4314Rights> getRights() {
     }
 
     @Override
-    public final boolean equals(Object o) {
+    public boolean equals(Object o) {
```

Comment:
Still valid regarding EqualsVerifier?

(equals and hashcode shall be final)

Reply:
the IDE advised against the final, arguing it was useless since the class is already `final`.
i think it doesn't make that big a difference. should i put it back ?

Reply:
same thing about all the other `final` i removed

Reply:
You should put it back.

https://jqno.nl/equalsverifier/manual/final/